packages/app/src/app/pages/Dashboard/Components/Branch/BranchCard.tsx

```diff
@@ -53,7 +51,7 @@ export const BranchCard: React.FC<BranchDetails> = ({
         align="center"
         justify="center"
       >
-        <Icon aria-hidden color="#808080" name="fork" size={40} />
+        <Icon aria-hidden color="#808080" name="branch" size={40} />
```

Comment:
```suggestion
        <Icon color="#808080" name="branch" size={40} />
```

Can be removed as you've added it to the `Icon` component yesterday 😃